packages/waku/tests/create-pages.test.ts

```diff
@@ -20,6 +19,7 @@ import { renderToString } from 'react-dom/server';
 import { expectType } from 'ts-expect';
 import type { TypeEqual } from 'ts-expect';
 import type { PathsForPages } from '../src/router/base-types.js';
+import { GetSlugs } from '../src/router/create-pages-utils/inferred-path-types.js';
```

Comment:
Isn't it `import type ...` ? I wonder why `"verbatimModuleSyntax": true,` doesn't complain.

Reply:
good catch, updated

It isn't complaining for the same reason that the import from the wrong file was not complaining I think... `tests` is not included in the tsconfig & the rootDir is `./src`

https://github.com/dai-shi/waku/blob/main/packages/waku/tsconfig.json#L8

I'll look into fixing this type checking. Basically what we want is for tsc to check the `tests` folder, but not output anything for it.

Reply:
> `tests` is not included in the tsconfig & the rootDir is `./src`

Ohhhhh, I see. cc @himself65

Reply:
Fixed with the latest commit! I'm happy to move things around or change the package json scripts changed as you'd like